### PR TITLE
Issue #7208: aligned javadoc/xdoc for JavadocParagraph

### PR DESCRIPTION
**Pull request by pbludov**
Issue #7208 (part of #5750)

No changes in the code.

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java [hunk 1]
```diff
@@ -26,49 +26,83 @@
 import com.puppycrawl.tools.checkstyle.utils.JavadocUtil;
 
 /**
- * Checks that:
+ * <p>
+ * Checks the Javadoc paragraph. Checks that:
```
Comment (pbludov) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java]: Need a summary that ends with period.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java [hunk 2]
```diff
@@ -26,49 +26,83 @@
...
+ * <p>
+ * To configure the default check:
  * </p>
  * <pre>
  * &lt;module name=&quot;JavadocParagraph&quot;/&gt;
  * </pre>
- *
- * <p>To allow newlines and spaces immediately after the &lt;p&gt; tag:
+ * <p>
+ * By default, the check will report a violation if there is a new line
```
Comment (pbludov) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java]: The `allowNewlineParagraph` property works the opposite way, so we need more samples here.

File: src/xdocs/config_javadoc.xml [hunk 3]
```diff
@@ -860,38 +862,54 @@ public int checkReturnTag(final int aTagIndex,
...
+&#47;**
+ * No tag (ok).
+ *
+ * &lt;p&gt;Tag immediately before the text (ok).
+ *
+ * &lt;p&gt;
+ * New line after tag (violation).
+ *
+ * &lt;p&gt; Whitespace after tag (violation).
+ *
```
Comment (romani) [inline src/xdocs/config_javadoc.xml]: please do us favor and add extra line to show that violation appear if no empty line between paragraphs.
`* &lt;p&gt; no empty line before p tag (violation).`

and the same line to the second example.

File: src/xdocs/config_javadoc.xml [hunk 4]
```diff
@@ -812,10 +812,10 @@ public int checkReturnTag(final int aTagIndex,
     </section>
 
     <section name="JavadocParagraph">
+      <p>Since Checkstyle 6.0</p>
       <subsection name="Description" id="JavadocParagraph_Description">
-        <p>Since Checkstyle 6.0</p>
         <p>
-          Checks that:
+          Checks the Javadoc paragraph. Checks that:
```
Comment (romani) [inline src/xdocs/config_javadoc.xml]: please make it `Checks Javadoc paragraphs be separated by empty line.`
To make sure that this Check is not responsible for all kind of validations over paragraph, as second validation is optional we can skip it from summary.

Comment (pbludov) [reply src/xdocs/config_javadoc.xml]: done

File: src/xdocs/config_javadoc.xml (hunk 3 above)
Comment (pbludov) [reply src/xdocs/config_javadoc.xml]: done

**Pull request merged**